.travis.yml

```
@@ -1,11 +1,36 @@
-language: java
-jdk:
-  - oraclejdk8
-install: true
-script : 
-  - mvn install -DskipTests
-  - mvn verify
-sudo: false
-addons:
-  sonarcloud:
-    organization: "lequal"
+language: java
+sudo: false
+install: true
+
+addons:
+  sonarcloud:
+    organization: "lequal"
+
+jdk:
+- openjdk8
+
+script:
+- mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar

```

Maybe you can do the following to fix travis build:
1/ add this section
```
services:
   - docker
```
2/
```suggestion
- docker run sonarqube:lts
- sleep 60 #Waiting for SonarQube
- mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar
```

or remove the `sonar:sonar` goal